Marlin/Marlin_main.cpp

```diff
@@ -1175,7 +1366,7 @@ inline void get_serial_commands() {
             #else
               safe_delay(1000);
             #endif
-            set_led_color(0, 0, 0);   // OFF
+			handle_led_print_event(all_off);
```

Comment:
Beware of tab characters. Please convert all indentation to use spaces, and remove any trailing whitespace also.